### PR TITLE
Reflect changes to the storage location of durability policies

### PR DESCRIPTION
**Pull request by GuptaManan100**
This PR makes changes to the docs to reflect the changes made in the following PRs- 
- https://github.com/vitessio/vitess/pull/10201
- https://github.com/vitessio/vitess/pull/10221
- https://github.com/vitessio/vitess/pull/10375
- https://github.com/vitessio/vitess/pull/10423

### CODE REVIEW

File: content/en/docs/14.0/user-guides/configuration-basic/durability_policy.md [hunk 1]
```diff
@@ -22,20 +22,20 @@ There are 2 implementations supported in this release:
 
 [EmergencyReparentShard](../../configuration-advanced/reparenting/#emergencyreparentshard-emergency-reparenting) and [PlannedReparentShard](../../configuration-advanced/reparenting/#plannedreparentshard-planned-reparenting) will use the durability rules while choosing the correct candidate for promotion.
 
-This configuration should be specified in [vtctld](../vtctld), [vtctl](../../../concepts/vtctl) as a flag `--durability_policy`. It should be specified in [vtorc](../vtorc) as `Durability` config.
+This configuration should be stored in the topo server in the keyspace record using the command [CreateKeyspace](../../../reference/programs/vtctl/keyspaces/#createkeyspace) or SetKeyspaceDurabilityPolicy.
```
Comment (GuptaManan100) [inline content/en/docs/14.0/user-guides/configuration-basic/durability_policy.md]: We have no docs for commands present in `vtctldclient`. We haven't added any equivalent of `SetKeyspaceDurabilityPolicy` to vtctl, so I don't know where to document this command

**Review by ajm188 — APPROVED** [review]

File: content/en/docs/14.0/user-guides/configuration-basic/durability_policy.md (hunk 1 above)
Comment (deepthi) [reply content/en/docs/14.0/user-guides/configuration-basic/durability_policy.md]: We now have docs here https://vitess.io/docs/14.0/reference/programs/vtctldclient/ so this PR can be updated.
You will also need to copy over all the documentation to 15.0

Comment (GuptaManan100) [reply content/en/docs/14.0/user-guides/configuration-basic/durability_policy.md]: Update the PR and also copied the changes to release-15.0

File: content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md [hunk 2]
```diff
@@ -22,20 +22,20 @@ There are 2 implementations supported in this release:
 
 [EmergencyReparentShard](../../configuration-advanced/reparenting/#emergencyreparentshard-emergency-reparenting) and [PlannedReparentShard](../../configuration-advanced/reparenting/#plannedreparentshard-planned-reparenting) will use the durability rules while choosing the correct candidate for promotion.
 
-This configuration should be specified in [vtctld](../vtctld), [vtctl](../../../concepts/vtctl) as a flag `--durability_policy`. It should be specified in [vtorc](../vtorc) as `Durability` config.
+This configuration should be stored in the topo server in the keyspace record using the command [CreateKeyspace](../../../reference/programs/vtctldclient/vtctldclient_createkeyspace/) or [SetKeyspaceDurabilityPolicy](../../../reference/programs/vtctldclient/vtctldclient_setkeyspacedurabilitypolicy/).
```
Comment (ajm188) [inline content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md]: "should be" or "must be" ?

File: content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md [hunk 3]
```diff
@@ -22,20 +22,20 @@ There are 2 implementations supported in this release:
...
 
 {{< info >}}
 Currently the durability policies are not used to setup semi-sync in EmergencyReparentShard or PlannedReparentShard. All the RPCs are still using the `--enable_semi_sync` flag on vttablet to setup semi-sync. This flag is currently being used for promotion rules and to log discrepancies in semi-sync setup. Nonetheless, this flag should be specified correctly for upgrade considerations to future releases when the durability policies will be used to setup semi-sync and `--enable_semi_sync` is deprecated.
 {{< /info >}}
 
-In case you notice any logs that look like the following, please check that your vtctld and vttablet configurations match:
+In case you notice any logs that look like the following, please check that your vttablet configuration and keyspace record in topo match:
```
Comment (ajm188) [inline content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md]:
```suggestion
If you notice any logs like the following, please verify that your vttablet configuration and keyspace record in topo match:
```

File: content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md [hunk 4]
```diff
@@ -22,20 +22,20 @@ There are 2 implementations supported in this release:
...
 ```
 invalid configuration - semi-sync should be setup according to durability policies, but enable_semi_sync is not set
 ```
 ```
 invalid configuration - semi-sync should be setup according to durability policies, but the tablet is not primaryEligible
 ```
-If `--enable_semi_sync` is set on the vttablets, then `semi_sync` durability policy should be used.  If semi-sync is not being used then `--durability_policy` should be set to `none`.
+If `--enable_semi_sync` is set on the vttablets, then `semi_sync` durability policy should be stored in the keyspace record. If semi-sync is not being used then the durability policy should be `none`.
```
Comment (ajm188) [inline content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md]: Same question about "should" vs "must"

**Review by ajm188 — APPROVED** [review]
couple suggestions! the "deploy-preview" action seems stuck, so i can't check that the backlinks are working properly, but i'm still going to approve

File: content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md (hunk 2 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md]: Fixed

File: content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md (hunk 4 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md]: Fixed

File: content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md (hunk 3 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/durability_policy.md]: Fixed

**Review by ajm188 — APPROVED** [review]

**Pull request merged**